man/shadow.5.xml

```diff
@@ -72,15 +72,15 @@
 	<term><emphasis role="bold">encrypted password</emphasis></term>
 	<listitem>
 	  <para>
-	    This field may be empty, in which case no passwords are
-	    required to authenticate as the specified login name.
-	    However, some applications which read the
-	    <filename>/etc/shadow</filename> file may decide not to permit
-	    any access at all if the password field is empty.
+	    This field may be empty, in which case
+	    no passwords are required to authenticate with the specified login name.
+	    However, some applications that read
+	    the <filename>/etc/shadow</filename> file may prevent any access
+	    if the password field is empty.
 	  </para>
 	  <para>
 	    A password field which starts with an exclamation mark means
-	    that the password is locked.  The remaining characters on the
+	    that the password is locked. The remaining characters on the
```

Comment:
Hi @domiborges !

This is a contentious thing.  The *correct* amount of inter-sentence space is two.  See some history about this:
<https://web.archive.org/web/20171217060354/http://www.heracliteanriver.com/?p=324>.

Don't let others lie to you about it; one space is wrong.  TL;DR: editorial companies moved from two spaces to one space to reduce costs, and be able to hire less experienced editors.  :-)

Reply:
On the other hand, semantic newlines preclude this at all.  After period, one should start a new source line.

Reply:
In the Linux man-pages project, we have two spaces as a rule, for commit messages (in the source code of manual pages we don't need this rule, due to semantic newlines).

Reply:
Indeed, two spaces.

Reply:
Updated.